content/introduction/why-dolt.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Why Dolt
+---
+
+## Background
+Existing relational database solutions came to market to provide a persistence layer for the multi-user applications the internet enabled.
+
+Their design goal was to capture the state of the application, and serve it up in a performant way.
+
+## Data Has Changed, Databases Have Not
+As adoption of software enabled business processes has exploded, so to have the ways in which we use data. It is extremely common for companies to have entire teams dedicated to data engineering, the practice of maintaining pipes to ensure these software enabled business process are supplied with the data they need to function properly.
```

Comment:
I don't like this lead in. I might move the observation that's what changed is "bad data can break your application" whereas before data was CRUD. It was consumed by humans who could apply judgment before use. Or something to that effect.